gazebo_ros/include/gazebo_ros/conversions/geometry_msgs.hpp

```diff
@@ -15,15 +15,16 @@
 #ifndef GAZEBO_ROS__CONVERSIONS__GEOMETRY_MSGS_HPP_
 #define GAZEBO_ROS__CONVERSIONS__GEOMETRY_MSGS_HPP_
 
+#include <ignition/math/Pose3.hh>
+#include <ignition/math/Quaternion.hh>
+#include <ignition/math/Vector3.hh>
+
 #include <geometry_msgs/msg/point.hpp>
 #include <geometry_msgs/msg/point32.hpp>
 #include <geometry_msgs/msg/pose.hpp>
 #include <geometry_msgs/msg/quaternion.hpp>
 #include <geometry_msgs/msg/transform.hpp>
 #include <geometry_msgs/msg/vector3.hpp>
-#include <ignition/math/Pose3.hh>
-#include <ignition/math/Quaternion.hh>
-#include <ignition/math/Vector3.hh>
```

Comment:
I guess cpplint thinks these are c-system headers? The other option is the use double-quotes for all includes not from the standard library. I don't have a preference either way though.

Reply:
I guess the new version of `cpplint` thinks that `.hh` headers are C headers instead of C++

I can switch to double quotes if you think that's a better option

Reply:
I don't have a preference; I don't think there's an objectively better reason to use double-quotes. If we use double-quotes, I think we can keep the `.hh` files _below_ any standard library headers :shrug:. The size of the diff might actually turn out to be larger if we switch to double-quotes.